include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -293,11 +293,51 @@ struct __parallel_scan_submitter;
 template <typename _CustomName, typename... _PropagateScanName>
 struct __parallel_scan_submitter<_CustomName, __internal::__optional_kernel_name<_PropagateScanName...>>
 {
+    static constexpr int
+    __iters_per_item(::oneapi::dpl::__internal::__algorithm_type __algo_type)
```

Comment:
We might want to think about how we wish to store tuning parameters and maybe consider finding a way to tag / label the specific architecture they are tuned for. In the future, we may have a new set of best performing magic numbers for a newer architecture and may need to choose which to use.